src/caw/caw_function.cc

```diff
@@ -173,4 +182,9 @@ void CawFunction::ReadReplys(const std::string& caw_id, KeyValueInterface& kv,
   }
 }
 
+std::vector<std::string> GetHashtags(const std::string& key) {
```

Comment:
maybe change `key` to `message` or `input`?

Reply:
Done, good call!